tests/conftest.py

```diff
@@ -19,6 +19,7 @@
     "DS_SLACK_WEBHOOK",
     "SE_SLACK_WEBHOOK",
     "DE_SLACK_WEBHOOK",
+    "FLORENCE_TOKEN",
```

Comment:
so this ought to blank any of these existing env var (where they exist) before the tests run and just for the length of the tests, so you shouldn't need to monkeypatch delete this in individual tests.